server/models/Appointment.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const appointmentSchema = new mongoose.Schema({
+  meeting_id: {
+    type: String,
+    required: true,
+    unique: true
```

Comment:
There could be multiple appointments involved in same meeting, you can remove _unique_